### DISCUSSION TITLE
Allow to control recording devices as well

### DISCUSSION THREAD

**Discussion by ViktorShapoval** [File-New-Project/EarTrumpet#1501] [Ideas]
**Recording devices management**

I would like to be able to control recording devices with eartrumpet as well as playback devices.

I made this small demo in code, changes are quite small since I'm simply reusing existing `DeviceCollectionViewModel` but initializing it with deviceManager created with `AudioDeviceKind.Recording`
![image](https://github.com/File-New-Project/EarTrumpet/assets/10196415/b80d9e6a-04b8-4933-8229-b795b83cb87d)

**Setting device as default for communications**

Another contribution that I would like to add is ability to make device default for both Multimedia and Recording at the same time. Code change will be small as well, something like this.
![image](https://github.com/File-New-Project/EarTrumpet/assets/10196415/110c9565-5fa4-4be0-bb0e-f5ff163dc53b)

Please tell me what do you think about both suggestions, I'm ready to contribute the code for them.
I saw there were a few discussions about both topics but they were suggestions with multiple other features, I want to keep it simple for my first contribution.

**Comment by Chrigi2486** [comment]
I think this is a great idea and definitely something I need.  
I'd recommend you check out [how to contribute](https://github.com/File-New-Project/EarTrumpet/blob/master/CONTRIBUTING.md) and I look forward to seeing these features in app in the future!
